lws_config.h.in

```diff
@@ -17,6 +17,12 @@
 /* The Libwebsocket version */
 #cmakedefine LWS_LIBRARY_VERSION "${LWS_LIBRARY_VERSION}"
 
+#define LWS_LIBRARY_VERSION_MAJOR ${LWS_LIBRARY_VERSION_MAJOR}
+#define LWS_LIBRARY_VERSION_MINOR ${LWS_LIBRARY_VERSION_MINOR}
+#define LWS_LIBRARY_VERSION_PATCH ${LWS_LIBRARY_VERSION_PATCH}
+/* LWS_LIBRARY_VERSION_NUMBER looks like 1005001 for e.g. version 1.5.1 */
+#define LWS_LIBRARY_VERSION_NUMBER (LWS_LIBRARY_MAJOR*1000000)+(LWS_LIBRARY_VERSION_MINOR*1000)+LWS_LIBRARY_VERSION_PATCH
```

Comment:
I prefer this format:
```
(LWS_LIBRARY_MAJOR << 16) | (LWS_LIBRARY_VERSION_MINOR << 8) | LWS_LIBRARY_VERSION_PATCH
````
And also adding a "release" flag at the end:

```
(LWS_LIBRARY_MAJOR << 24) | (LWS_LIBRARY_VERSION_MINOR << 16) | (LWS_LIBRARY_VERSION_PATCH << 8) | LWS_LIBRARY_VERSION_RELEASE
```

If `LWS_LIBRARY_VERSION_RELEASE` is set to `0` in the `libwebsockets.h` header, this means it is an official release. Normally is set to `1` to signify a beta version...